assets/dev/js/editor/elements/views/base.js

```diff
@@ -277,6 +278,34 @@ BaseElementView = BaseContainer.extend( {
 		}
 	},
 
+	updateHandlesOverlay() {
+		const $overlayList = this.$el.find( '.elementor-editor-element-settings' );
+
+		const elementStyles = window.getComputedStyle( this.el ),
+			elementType = this.$el.data( 'element_type' ),
+			overflowStyles = [ elementStyles.overflowX, elementStyles.overflowY, elementStyles.overflow ],
+			isHaveOverflow = overflowStyles.includes( 'hidden' ) || overflowStyles.includes( 'auto' ),
+			isContainer = getAllElementTypes().includes( elementType );
```

Comment:
I think the name is a bit misleading, it's an element, not necessarily a container, isn't it?

```suggestion
			isElement = getAllElementTypes().includes( elementType );
```